gnosis/verification.k

```diff
@@ -43,10 +43,11 @@ module VERIFICATION
 
     // select/store key equality
 
-    rule hash2(_,_) ==K I => false
-      requires #isConcrete(I)
+    rule hash2(A,B) ==K I => false
+      requires notBool #isConcrete(hash2(A,B)) andBool #isConcrete(I)
+      				 andBool I <Int 1000000000 //random small-ish number bigger than any fixed storage location
```

Comment:
not sure if 1000000000 is small enough. could you make it tighter?

Reply:
Why not:) Just pick a value. But you got the point of this side condition.
10^9 ~=2^30 is still tens of orders of magnitude less than 2^256 > 10^75, are you afraid it's so big it could lead to collision:) ?

Reply:
well, we're talking about probability. you're assuming that the probability of getting a hash value that has ~220 leading zeros is negligible. someone agrees on that. but i'm saying that it is better to make the probability even smaller.  i mean, i think `I` that appears in the execution of gnosis should be less than 10 or 20, since they are all storage index for the global variables, right?  btw, it's not urgent, and we can do this later.

Reply:
fixed